ECommerceAPI/Common/Constants.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ECommerceAPI.Common
+{
+    public static class Constants
+    {
+        public static string ResetPasswordUrl = "https://localhost:7131/reset-password";
```

Comment:
please try to think, if you published this project to hosting how you will handle the URL? in net we have something can define the environment is debug or release to define the URL try to search about it